tests/integration/telemetry/tracing/otelcollector/testdata/otel-tracing-res-detectors.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  name: otel-tracing
+spec:
+  tracing:
+  - providers:
+    - name: test-otel-res-detectors
+    randomSamplingPercentage: 100.0
```

Comment:
In the PR we discussed the case of a user setting sampling to not 100% we would override it. I don't see that implemented, though

Reply:
This PR is just about resource detectors. Sampling will come in a separated PR.

Reply:
oops, thanks! got them mixed up

Reply:
No prob!